frontend/src/types.ts

```diff
@@ -608,13 +608,15 @@ export interface MatchedRecording {
 }
 
 interface CommonActorType {
-    id?: string | number
+    id: string | number
     properties: Record<string, any>
-    created_at?: string
-    matched_recordings?: MatchedRecording[]
+    created_at: string
+    matched_recordings: MatchedRecording[]
+    value: number | null
```

Comment:
It should be _somewhat_ generic… But I see how this can be too much.
With `series_value`, this wouldn't necessarily be correct, since this isn't graphable data. `aggregation_value` makes sense, but we already use `aggregated_value` for the total value of an insight series (i.e. the value for all time series points together). Went for `value_at_data_point`.